client/src/components/PlanShowPage.js

```diff
@@ -1,7 +1,10 @@
+import { pullAllBy } from 'lodash';
```

Comment:
two things:
- I dont actually see you using this function?
- convention for lodash is to always do `import _ from "lodash"`. then do `_.pullAllBy()`